pandas/core/internals/managers.py

```diff
@@ -566,8 +566,20 @@ def where(self, **kwargs) -> "BlockManager":
     def setitem(self, indexer, value) -> "BlockManager":
         return self.apply("setitem", indexer=indexer, value=value)
 
-    def putmask(self, **kwargs):
-        return self.apply("putmask", **kwargs)
+    def putmask(
+        self, mask, new, align: bool = True, axis: int = 0,
+    ):
+        transpose = self.ndim == 2
+
+        return self.apply(
+            "putmask",
+            mask=mask,
+            new=new,
+            align=align,
+            inplace=True,
```

Comment:
How feasible is it to align the inplace arg with blocks?

Reply:
Block.putmask is called from other Block methods with inplace=False, so non-trivial